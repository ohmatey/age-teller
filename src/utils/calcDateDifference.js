const calcDateDifference = date => {
  const today = new Date().getTime()
  const comparisonDate = new Date(date).getTime()
  const difference = Math.abs(today - comparisonDate)
  const days = Math.floor(difference / ( 1000 * 3600 * 24 ))

  return `You were born ${days} days ago!`
}

export default calcDateDifference
