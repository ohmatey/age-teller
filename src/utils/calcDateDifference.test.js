import calcDateDifference from './calcDateDifference'

test('calcDateDifference function returns expected string of date values', () => {
  expect(calcDateDifference('2017-06-28')).toMatchSnapshot()
})
