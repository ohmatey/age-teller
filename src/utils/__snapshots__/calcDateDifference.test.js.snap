// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`calcDateDifference function returns expected string of date values 1`] = `"You were born 2 days ago!"`;
